packages/cf-style-container/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "css-color-function": "^1.3.0",
+    "fela": "^4.3.2",
```

Comment:
what's the difference between `cf-style-container` and `cf-style-provider`. To me they could be merged into single component.